script/tool/lib/src/xctest_command.dart

```diff
@@ -164,6 +107,31 @@ class XCTestCommand extends PluginCommand {
     }
   }
 
+  Future<int> _runTests(bool runTests, String destination, Directory example) {
+    final List<String> xctestArgs = <String>[
+      _kXcodeBuildCommand,
+      if (runTests)
+        'test',
+      'analyze',
+      '-workspace',
+      'ios/Runner.xcworkspace',
+      '-configuration',
+      'Debug',
+      '-scheme',
+      'Runner',
```

Comment:
I didn't know this, making the scheme to be the app target runs all the test targets?

Reply:
No, it runs whatever tests are hooked up in the specified scheme. So in this case, `xcodebuild test -scheme Runner` will run the tests set up in the `Runner` scheme.
![Screen Shot 2021-03-02 at 5 52 00 PM](https://user-images.githubusercontent.com/682784/109740369-067fc800-7b80-11eb-9ad5-f6fd48d6c52d.png)